guidelines/taxation.md

```diff
@@ -0,0 +1,41 @@
+# Taxation
+
+## Legal environments, tax environments, and tax codes in Mews
```

Comment:
Why do we mention legal environments here if the rest of the article doesn't refer to them at all? Or actually, it refers to them but it seems to be mixing legal envs. and tax envs.